action.yml

```diff
@@ -91,6 +91,9 @@ inputs:
     description: |
       Extra environment variables to set for the speakeasy run execution.
     required: false
+  pnpm_version:
```

Comment:
Should need to also add this to workflow-executor.yaml otherwise they won't be able to pass it through

Reply:
Should be covered by https://github.com/speakeasy-api/sdk-generation-action/pull/168/commits/79aeb68c7e7cd770737cfdb3f93d98e2493908d2 -- please let me know if there's anything else there

Reply:
needs to also add inputs.pnpm_version at the bottom of this file